tests/improver-nowcast-extrapolate/01-help.bats

```diff
@@ -63,6 +64,9 @@ optional arguments:
   --northward_advection_filepath NORTHWARD_ADVECTION_FILEPATH
                         Path to input file containing Northward advection
                         velocities.
+  --json_file JSON_FILE
```

Comment:
Should probably also link to where in the code they can look up the form of the json file.

Reply:
Added.